tests/unit/s2n_tls13_server_cert_test.c

```diff
@@ -25,6 +25,33 @@
 #include "utils/s2n_safety.h"
 
 /* Test vectors from https://tools.ietf.org/html/rfc8448#section-3 */
+
+/* whole cert message without 0b0001b9 header */
+const char tls13_cert_message_hex[] =
```

Comment:
is this a merge conflict from before, or are we able to concatenate the following parts to form this?

Reply:
Yes. And the second test had previously been implemented to treat the whole chunk after the header (cert+extensions) as the cert. As the two tests were treating the sample cert message differently, I opted to keep them separate in my fix to try to make that more clear. I definitely think it makes the first test cleaner. But if you think it's cleaner overall to just do concatenations for both, I can change that